protocol/messenger.go

```diff
@@ -4716,19 +4716,26 @@ func (m *Messenger) MuteChat(chatID string) error {
 
 	var contact *Contact
 	if chat.OneToOne() {
-		contact, _ = m.allContacts.Load(chatID)
+		contact, _ = m.allContacts.Load(MuteParams.ChatID)
 	}
 
-	return m.muteChat(chat, contact)
+	return m.muteChat(chat, contact, MuteParams.MutedAt, MuteParams.MutedType)
+}
+
+func (m *Messenger) MuteChatV2(chatID string, mutedAt int64, muteType int) error {
```

Comment:
You can use a single map parameter and add it to `protocol/requests`

Reply:
Well, We definitely can .. What do you prefer to be sent over the wire from status-mobile to status-go .. maps or values ?

Reply:
And if it's a map .. Will I send it as a javascript object or a CLJ map?

Reply:
I, for one, much prefer maps in `status-go` endpoints because they're open and extensible. Endpoints with positional arguments can't evolve their signature without causing breaking changes on clients.

>And if it's a map .. Will I send it as a javascript object or a CLJ map?

In Mobile you just send a CLJ map in the `json-rpc/call` effect and it'll be correctly serialized for status-go.